Model/CreditCardPaymentMethod.php

```diff
@@ -232,7 +232,7 @@ public function getConfigFields()
             ],
         ];
 
-        return array_merge($parentConfigFields, $additionalFields);
+        return array_merge($aParentConfigFields, $aAdditionalFields);
```

Comment:
you always use the + please update it to '+' or all the others to array_merge. Maybe this would also be an additional rule